client/src/components/WishItem/WishItem.module.scss

```
@@ -0,0 +1,103 @@
+@import "../../global.scss";
+
+.wishItem {
+    background: rgb(154, 231, 157);
+    background: linear-gradient(47deg, rgba(154, 231, 157, 1) 0%, rgba(255, 255, 255, 1) 35%, rgba(172, 244, 189, 1) 83%);
+    border-radius: 2vw;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 30% 35% 35%;
+    align-items: center;
+
+    img {
+        width: 100%;
+        aspect-ratio: 1/1;
+        border-radius: 2vw;
```

Das ist extrem seltsam. Soll der Border radius anders sein, je nach dem auf welchem Gerät du bist? 
```suggestion
        border-radius: 8px;
```